src/main/java/co/inventrosoft/springboottask/repository/TeamRepositoryFileImpl.java

```diff
@@ -0,0 +1,74 @@
+package co.inventrosoft.springboottask.repository;
+
+
+import co.inventrosoft.springboottask.model.Team;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+@Qualifier
```

Comment:
Why do you need it?
If you need specific implementation to be injected, use injection by type (not by interface).